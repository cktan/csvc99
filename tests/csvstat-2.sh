# Test Case : Handling of Special Characters
# 	 quotes, commas, newlines within fields
../csvstat in/csvstat-2.csv 
