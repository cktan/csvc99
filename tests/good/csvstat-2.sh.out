      #bytes: 104
       #rows: 3
    #columns: 3
avg row size: 34
min row size: 25
max row size: 40
